$OpenBSD$

Increase the thread stack size to the default size on Linux, fixes
a crash.

--- src/sshmasterconnection.cpp.orig	Tue Jul 28 04:50:35 2015
+++ src/sshmasterconnection.cpp	Sun May 15 02:53:37 2016
@@ -152,7 +152,7 @@ SshMasterConnection::SshMasterConnection (QObject* par
         QString proxylogin, QString proxypassword, QString proxykey,
         bool proxyautologin, bool proxyKrbLogin ) : QThread ( parent )
 {
-#if defined ( Q_OS_DARWIN )
+#if defined ( Q_OS_DARWIN ) || defined(__OpenBSD__)
     // Mac OS X provides only 512KB stack space for secondary threads.
     // As we put a 512KB buffer on the stack later on, we need a bigger stack space.
     setStackSize (sizeof (char) * 1024 * 1024 * 2);
